common/src/test/java/com/microsoft/identity/common/integration/ClientCredentialsGrant/OAuth2/MicrosoftSTSClientCredentialsGrantTest.java

```diff
@@ -68,7 +69,7 @@ public void test_ClientCredentials() throws CertificateException, UnrecoverableK
 
         MicrosoftClientAssertion assertion = new MicrosoftClientAssertion(audience, credential);
 
-        TokenRequest tr = new TokenRequest();
+        TokenRequest tr = new MicrosoftStsTokenRequest();
```

Comment:
This was previously failing on `strategy.validateTokenRequest()` with a `ClassCastException`